docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,177 @@
+
+#### Deployment configuration
+
+A [dagster_cloud.yml configuration file](/dagster-cloud/managing-deployments/dagster-cloud-yaml) in the root of the repository can be used to expose each project to Dagster Cloud as a distinct code repository.
+
+### Multiple VCS repositories (git, hg, etc.)
+
+For stronger isolation, you can use multiple VCS repositories to manage multiple projects.
+
+Pro:
+
+- stronger isolation between projects and teams
+- each project has its own CI/CD pipeline and be deployed independently
+- dependencies between projects can be managed independently
+
+Con:
+
+- code sharing between projects require additional coordination to publish and reuse packages between projects
+
+#### Deployment configuration
+
+Each project can be exposed to Dagster Cloud as a distinct code repository. For each repository, follow the [steps appropriate to your CI/CD provider](https://docs.dagster.io/dagster-cloud/getting-started#step-4-configure-cicd-for-your-project) and include only the code locations that are relevant to the repository in your CI/CD workflow.
+
+Example with GitHub CI/CD on Hybrid deployment:
+
+1. For each VCS repository, use the CI/CD workflow provided in [Dagster Cloud Hybrid Quickstart](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/.github/workflows/dagster-cloud-deploy.yml).
+
+2. For each project in the repository, you can configure the [code location to deploy in the `dagster_cloud.yml` file](/dagster-cloud/managing-deployments/dagster-cloud-yaml) and configure it accordingly to your project structure.
+
+   ```yaml
+   # File: dagster_cloud.yml
+   locations:
+     - location_name: project_a
+       code_source:
+         package_name: project_a
+       build:
+         # ...
+     - location_name: project_b
+       code_source:
+         package_name: project_b
+       build:
+         # ...
+   ```
+
+3. Modify the CI/CD workflow to deploy every code locations for this repository.
+
+   ```yaml
+   # File: .github/workflows/dagster-cloud-deploy.yml
+   # ...
+   jobs:
+     dagster-cloud-deploy:
+       # ...
+       steps:
+         - name: Update build session with image tag for "project_a" code location
+           id: ci-set-build-output-project-a
+           if: steps.prerun.outputs.result != 'skip'
+           uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
+           with:
+             command: "ci set-build-output --location-name=project_a --image-tag=$IMAGE_TAG"
+
+         - name: Update build session with image tag for "project_b" code location
+           id: ci-set-build-output-project-b
+           if: steps.prerun.outputs.result != 'skip'
+           uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
+           with:
+             command: "ci set-build-output --location-name=project_b --image-tag=$IMAGE_TAG"
+         # ...
+   ```
+
+## Strategies for execution context isolation
```

Comment:
possibly should be in a separate page?

Reply:
I would support that - there's a little bit of overlap here with some of the content in https://docs.dagster.io/dagster-cloud/deployment/agents/running-multiple-agents#running-multiple-agents

Reply:
I will wait on @slopp and @erinkcochran87 feedback and edit she offered to submit herself to reduce back and forth before changing structure further.

About "running multiple agents": I do mention the concept then link to that page further down, instead of duplicating explanations. What would you suggest that I change?